g0v.network./polis-explorations.g0v.network.yml

```
@@ -0,0 +1,18 @@
+---
+polis-explorations:
+  - type: TXT
+    value: 301 https://www.youtube.com/watch?v=pVJE6GXqzsw&list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```

```suggestion
    value: 301 https://www.youtube.com/playlist?list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```

good call @ballPointPenguin